### ISSUE TITLE
Package does not update via Nuget, have to install new each time

### ISSUE THREAD

**Issue by codereflection** [codereflection/Giles#24]
When an update is released for Giles, the package manager does not show that an update is available. If you install the new version, the old version of Giles is left in the packages folder.

Also, with multiple versions of Giles in the packages folder, the Powershell launch scripts will not always launch the newest version. Actually, the script may never launch the newer version, it all depends on how the directories are returned to the script from the file system, which may or may not be in order.

**Comment by Ang3lFir3** [comment]
install using -x and don't let those pesky folder names get you down! or use Chewie

**Comment by codereflection** [comment]
True, but I need to find a solution that works for everyone. Most people will be using the package manager from Visual Studio, and that's the only way that the launch scripts get installed out to the root of the solution.

**Comment by Ang3lFir3** [comment]
i wonder if Giles might be better suited for a choclatey distribution and not so much nuget ?? (still uses nuget... just a thought)

**Comment by codereflection** [comment]
That's also planned, see GH-17.

**Comment by motowilliams** [comment]
I also found that the powershell giles launch scripts don't get restored to the solution root folder when using Nuget Package Restore.  The package contents get restored just fine just not the giles-x86.ps1 and giles.ps1 files.
